src/sessions.ts

```diff
@@ -435,7 +435,7 @@ export class ClientSession extends TypedEventEmitter<ClientSessionEvents> {
   /**
    * Starts a transaction and runs a provided function, ensuring the commitTransaction is always attempted when all operations run in the function have completed.
    *
-   * **IMPORTANT:** This method requires the user to return a Promise, and `await` all operations.
+   * **IMPORTANT:** This method requires the function passed in to return a Promise. That promise must be made by `await`-ing all operations in such a way that rejections are propagated to the returned promise.
    *
    * @remarks
    * This function:
```

Comment:
One more fix up, since we are here, "this function" doesn't make sense for the bullets below this except for the last one. Maybe at some point they were phrased differently. Can you delete "This function:" and rephrase the last bullet. "If the driver needs... it may be called..."